src/CriticalBackgroundService.cs

```diff
@@ -42,12 +42,7 @@ protected CriticalBackgroundService(IApplicationEnder applicationEnder)
         /// but you can override it, if you want to handle it differently, or do some extra logging.
         /// </summary>
         /// <param name="exceptionFromExecuteAsync"></param>
-        protected virtual void OnError(Exception exceptionFromExecuteAsync)
-        {
-            Console.Error.WriteLine($"Error happened while executing CriticalBackgroundTask {this.GetType().FullName}. Shutting down.");
-            Console.Error.WriteLine(exceptionFromExecuteAsync.ToString());
-            this._applicationEnder.ShutDownApplication();
-        }
+        protected abstract void OnError(Exception exceptionFromExecuteAsync);
```

Comment:
I don't think I'm alright with this change - see #24

Reply:
Why?